pkg/manifestcomparators/comp_must_not_exceed_cost_budget.go

```diff
@@ -165,7 +203,7 @@ func getCostErrorMessage(costName string, expressionCost, costLimit uint64) stri
 
```

Comment:
```suggestion
return fmt.Sprintf("%s exceeds budget by factor of %s (try simplifying the rule(s), or adding maxItems, maxProperties, and maxLength where arrays, maps, and strings are declared)", costName, factor)
```

would make more sense for the total cost